Neos.Flow/Classes/Mvc/Routing/RouterCachingService.php

```diff
@@ -116,6 +117,8 @@ public function storeMatchResults(RouteContext $routeContext, array $matchResult
         if ($this->containsObject($matchResults)) {
             return;
         }
+
+        $httpRequest = $routeContext->getHttpRequest();
```

Comment:
You wanted to do something with the http request here?

Reply:
Merge error, I had originally used it in the next line but you extract it directly, so will be removed.